src/vs/workbench/browser/parts/views/media/views.css

```diff
@@ -9,40 +9,55 @@
 }
 
 .file-icon-themable-tree .monaco-tree-row .content::before {
-	background-size: 16px;
-	background-position: 50% 50%;
-	background-repeat: no-repeat;
-	padding-right: 6px;
+	-webkit-mask-size: 16px;
+	-webkit-mask-position: 50% 50%;
+	-webkit-mask-repeat: no-repeat;
+	-webkit-mask-image: url('');
+	margin-right: 6px;
 	width: 16px;
 	height: 22px;
 	display: inline-block;
 	vertical-align: top;
 	content: ' ';
 }
 
+/* Default Icon Colors */
+.vs-dark .monaco-tree-row .content::before {
```

Comment:
I would prefer if this moved into a theming participant so that our CSS stays clean of any color definitions. Otherwise it is hard to track which color is truly dynamic.

Reply:
Removed these css defaults and added support to inherit the default foreground color from the color registry.